nl-be/auth.json

```diff
@@ -0,0 +1,284 @@
+{
+    "spec": {
+      "language": "nl-be",
```

Comment:
@onderay sorry to be pedantic but can we make this property and also the folder name `nl-BE`.

We'll also need to add it to the `readme.md` as it isn't listed there with all the other codes, @evgenyk do we need to do something to get this added in the app code also?

Reply:
https://github.com/kinde-oss/kinde-translations#language-codes is where I meant